libs/features/mas/src/literals.js

```diff
@@ -1,5 +1,7 @@
 import { Defaults } from './defaults.js';
-import { equalsCaseInsensitive } from './external.js';
+import { equalsCaseInsensitive } from '@dexter/tacocat-core';
```

Comment:
i live it to your preferrence but imo we should remove whole @dexter/tacocat-core package.
you could copy over required methods (i am not sure if all of them are indeed need to be 'reused') to utilities and remove unneccessary package dependency. we only use tacocat-core for basic js functions, but it brings useless code as well